### PR TITLE
Fixed comments for execute.as.user changes (#807)

### PR DESCRIPTION
**Pull request by jamiesjc**
Removed a private function to make the code easier to read.
Added comments to the changes in test cases.
Abandoned previous pull request (#807).

### CODE REVIEW

File: azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java [hunk 1]
```diff
@@ -114,7 +114,11 @@ public void setUp() throws IOException {
     props.put(CommonJobProperties.JOB_ID, "test_job");
     props.put(CommonJobProperties.EXEC_ID, "123");
     props.put(CommonJobProperties.SUBMIT_USER, "test_user");
-    props.put("execute.as.user", "false");
+
```
Comment (HappyRay) [inline azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java]: Consider consolidating the common code in a common setup method in tests?

**Comment by jamiesjc** [comment]
The configuration for props doesn't seem to be common for most test cases. Even for these two similar test cases, they have different props like "type". It might not be necessary to create a new common setup method?

**Comment by fsi206914** [comment]
@HappyRay @jamiesjc  Program readability is also important to readers. I don't have preferences given this case.

**Comment by HappyRay** [comment]
@jamiesjc please update the documentation as needed to reflect the new default. 
When we release a new version, we need to highlight this change. A user who upgrades will need to know the new behavior and adjust their configs accordingly..

**Comment by fsi206914** [comment]
Confused if this change is merged or not. @jamiesjc

**Pull request closed without merging**